datacube/model/__init__.py

```diff
@@ -639,6 +642,20 @@ def validate_eo3(cls, doc) -> None:
         cls.validate(doc)  # type: ignore[attr-defined]
         validate_eo3_compatible_type(doc)
 
+    # Fields defined using SQLA ORM are not pickleable, so use a named (and therefore pickleable) extraction function
```

Comment:
Line looks really long, can this be broken to fit in 88 columns?

And I think "SQLA" will get some annoying squiggly lines in PyCharm, so great if SQLAlchemy can be spelled out. (Almost all my documentation fixes are because of squiggly lines in PyCharm so it finds real issues, so I want to keep that inspection enabled.)